sample-queries/sample-queries.json

```diff
@@ -2981,5 +2981,190 @@
+        "skipTest": false 
+    },
+    {
+        "id": "4a8fb118-f650-4efc-b109-d5dd4b3e9922",
+        "category": "Microsoft Teams (beta)",
+        "method": "DELETE",
+        "humanName": "delete channel",
+        "requestUrl": "/beta/teams/{team-id}/channels/{channel-id}",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-delete?view=graph-rest-beta",
+        "tip": "This query requires a team id and a channel id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams. To find the channel id of channels you belong to, stay in user (delegated permissions) mode and run: GET https://graph.microsoft.com/beta/teams/{team-id}/channels.",
+        "skipTest": false
+    },
+    {
+        "id": "44a072cc-c5c6-40f2-ab21-e7b0c417bb00",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "channels of a team which I am member of",
+        "requestUrl": "/beta/teams/{team-id}/channels",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-list?view=graph-rest-beta",
+        "tip": "This query requires a team id.  To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams",
+        "skipTest": false
+    },
+    {
+        "id": "3142bf45-6ab7-4008-a2a4-3c7555773cef",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "resource-specific permissions in chat",
+        "requestUrl": "/beta/chats/{chat-id}/permissionGrants",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-list-permissiongrants?view=graph-rest-beta&tabs=http",
+        "tip": "This query requires a chat id. To find the chat ids, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/beta/me/chats/",
+        "skipTest": false
+    },
+    {
+        "id": "fece537d-55e0-42d2-bcfd-e71674322967",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "app installations within a chat",
+        "requestUrl": "/beta/chats/{chat-id}/installedApps",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-list-installedapps?view=graph-rest-beta&tabs=http",
+        "tip": "This query requires a chat id. To find the chat ids, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/beta/me/chats/",
+        "skipTest": false
+    },
+    {
+        "id": "da59a1e1-4979-485e-a128-44f28d6d67ac",
+        "category": "Microsoft Teams",
+        "method": "GET",
+        "humanName": "get list of members in team",
+        "requestUrl": "/v1.0/teams/{team-id}/members",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/team-list-members?view=graph-rest-1.0&tabs=http",
```

Comment:
Shouldn't this be beta?

Reply:
RSC permissions for it is in production, since its the only sample query right now is in production; is that enough of a reason to put it in beta?

Reply:
Uhh, intersting, I'm not sure. If that's the case and you think it should stay as v1, that's fine